src/lib/components/steppers/Steppers.component.js

```diff
@@ -43,6 +47,11 @@ const Circle = styled.div`
         background-color: ${brandingTheme.primary};
         color: ${defaultTheme.white};
       `;
+    } else if (props.error) {
+      return css`
+        background-color: red;
+        color: ${defaultTheme.white};
```

Comment:
`${props => mergeTheme(props.theme, defaultTheme).text};` to support dark mode

Reply:
@carlito-scality but if the background for error is always red, white wouldn't be a better color here?